src/AutoBuild.jl

```diff
@@ -479,6 +479,7 @@ function register_jll(name, build_version, dependencies, julia_compat;
     end
     # Calculate tree hash of wrapper code
     wrapper_tree_hash = bytes2hex(Pkg.GitTools.tree_hash(code_dir))
+    wrapper_commit_hash = strip(read(`git -C $(code_dir) rev-parse HEAD`, String))
```

Comment:
```suggestion
    wrapper_commit_hash = readchomp(`git -C $(code_dir) rev-parse HEAD`)
```
is simpler, but why not using LibGit2 as we do everywhere else here?

Reply:
Done, take a look.